src/h5web/providers/hsds/models.ts

```diff
@@ -11,6 +9,72 @@ import type {
 } from '../hdf5-models';
 import type { Entity } from '../models';
 
+/* --------------------- */
+/* ----- RESPONSES ----- */
+
+export interface HsdsRootResponse {
+  root: HDF5Id;
+}
+
+export interface HsdsGroupResponse {
+  id: HDF5Id;
+  linkCount: number;
+  attributeCount: number;
+}
+
+export interface HsdsDatasetResponse {
+  id: HDF5Id;
+  shape: HsdsShape;
+  type: HsdsType;
+  attributeCount: number;
+}
+
+export interface HsdsDatatypeResponse {
+  id: HDF5Id;
+  type: HsdsType;
+}
+
+export interface HsdsAttributesResponse {
+  attributes: Omit<HsdsAttributeWithValueResponse, 'value'>[];
```

Comment:
I moved things around, so the diff is ugly, but here I'm now using `HsdsAttributeWithValueResponse` instead of `HDF5Attribute`, which is what allows me to remove the latter.

Reply:
Could we change the clunky `HsdsAttributeWithValueResponse` into `HsdsAttributeReponse` ?